### ISSUE TITLE
TWR SoftAP password not set or too short

### ISSUE THREAD

**Issue by sysadminpower2019** [ohanssen/ArcticTracker-ESP32#7]
Hello,

thanks for the project unfortunately the documentation is a bit unclear and confusing.
Why not just offer an AP with a default username / password combo that can be later changed.

Getting the radio to connect to the serial terminal can be a bit unreliable.
I tried to use api-key and provide a string of letter but that doesnt seem to be enough.

**Comment by ohanssen** [comment]
Yes, the documentation is unfortunately rather "limited" at the moment. There is some work to do. It might be a good idea to start with a default username/password for the soft AP. To be able to connect to the tracker via the web-ui you need to set the 'api-key' but also the 'api-origins'. Set it to '.*'. In the next release this will be the default.

**Comment by sysadminpower2019** [comment]
hello, thanks for your answer

yea the ideal would be able to flash the firmware and be able to connect
directly to the tracker without a laptop.
One question, does this project support sending both voice and aprs?
The other twr plus project allows both but unfortunately, has not been
updated for a while and is very unreliable.
However the concept of hitting the ptt key and sending voice and after an
aprs packet is a great idea.

**Comment by ohanssen** [comment]
Well, to flash it the first time you will need a computer and a USB cable. Currently, you will need to connect to the command-shell the first time as well to do the initial config. 

No the Arctic Tracker is just an APRS tracker. It started with my own hardware and was extended to run on the T-TWR as well.

**Comment by ohanssen** [comment]
Actually, it is possible to activate the soft-AP and connect it from the webapp (or a smartphone app). I'll look at some default settings to allow it to be used without needing to connect the command shell the first time.
